### PR TITLE
[9.0][MIG][website_sale_checkout_country_vat] Migrate.

### PR DESCRIPTION
**Pull request by yajo**
Migrated to v9, with corresponding changes.

@Tecnativa

### CODE REVIEW

**Review by lasley — APPROVED** [review]
Looks good 👍 

Also TIL `Website Snippet Country Code Dropdown` - spiffy

File: website_sale_checkout_country_vat/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+If you want to have a sane default also in the country address, check the
+`website_sale_default_country
+<https://github.com/OCA/e-commerce/tree/8.0/website_sale_default_country>`_
+module.
+
+Usage
+=====
+
+when a customer makes a purchase and do the checkout process the VAT country
```
Comment (pedrobaeza) [inline website_sale_checkout_country_vat/README.rst]: s/when/When

Comment (pedrobaeza) [reply website_sale_checkout_country_vat/README.rst]: comma after `checkout process`

File: website_sale_checkout_country_vat/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+code can be selected separated from the code itself.
```
Comment (pedrobaeza) [inline website_sale_checkout_country_vat/README.rst]: You can add:

```
This is useful in 2 ways:

- User is implicitly forced to input (select) a country with its code.
- Error in the VAT now only can be raised by an incorrect vat, not the absence of the country code, which usually confuses user.
```

File: website_sale_checkout_country_vat/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Website Sale Checkout Country VAT",
+    "summary": "Autocomplete VAT in checkout process",
+    "version": "9.0.1.0.0",
+    "category": "Website",
```
Comment (pedrobaeza) [inline website_sale_checkout_country_vat/__openerp__.py:10]: Isn't a specific category for e-commerce? We should create it if not.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You should squash also in pairs the two 8.0 commits from you, the OCA transbot and the removal of the en language, and finally your two 9.0 commits (till now). You can leave separated for now the commit amending these review, but we should squash it at the end also. The last 2 can be leaved until the "final squash" before merging.

File: website_sale_checkout_country_vat/__openerp__.py (hunk 3 above)
Comment (yajo) [reply website_sale_checkout_country_vat/__openerp__.py:10]: [`website_sale` has this](https://github.com/odoo/odoo/blob/9.0/addons/website_sale/__openerp__.py#L3)...

**Comment by yajo** [comment]
Readme improved, commits logically squashed.

File: website_sale_checkout_country_vat/__openerp__.py (hunk 3 above)
Comment (pedrobaeza) [reply website_sale_checkout_country_vat/__openerp__.py:10]: But as said, we are creating a new whole set of modules that should have an specific category although the main one is not differentiated.

Comment (pedrobaeza) [reply website_sale_checkout_country_vat/__openerp__.py:10]: Let's go this way for now, as there's no convention on this on the rest of the modules.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**